cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -29,18 +37,33 @@
 """
 
 minimal_ptx_kernel = """
-.version 8.5
-.target sm_90
-.address_size 64
-
 .func _MinimalKernel()
 {
     ret;
 }
 """
 
-ptx_kernel_bytes = ptx_kernel.encode('utf-8')
-minimal_ptx_kernel_bytes = minimal_ptx_kernel.encode('utf-8')
+ptx_kernel_bytes =  [(ptx_header(version, arch) + ptx_kernel).encode('utf-8') for version, arch in zip(PTX_VERSIONS, ARCHITECTURES)]
+minimal_ptx_kernel_bytes = [(ptx_header(version, arch) + minimal_ptx_kernel).encode('utf-8') for version, arch in zip(PTX_VERSIONS, ARCHITECTURES)]
+
+
+#create a valid LTOIR input for testing
+def CHECK_NVRTC(err):
+    if err != nvrtc.nvrtcResult.NVRTC_SUCCESS:
+        raise RuntimeError('Nvrtc Error: {}'.format(err))
+    
+empty_cplusplus_kernel = "__global__ void A() {}"
+err, program_handle = nvrtc.nvrtcCreateProgram(empty_cplusplus_kernel.encode(), b"", 0, [], [])
+CHECK_NVRTC(err)
+nvrtc.nvrtcCompileProgram(program_handle, 1, ["-dlto".encode()])
+err, size = nvrtc.nvrtcGetLTOIRSize(program_handle)
+CHECK_NVRTC(err)
+empty_kernel_ltoir = b" " * size
+err, = nvrtc.nvrtcGetLTOIR(program_handle, empty_kernel_ltoir)
+CHECK_NVRTC(err)
+err, = nvrtc.nvrtcDestroyProgram(program_handle)
+CHECK_NVRTC(err)

```

Comment:
Let's wrap these as a fixture so that they are not run at test collection time, something like
```suggestion
@pytest.fixture
def get_dummy_ltoir():
    def CHECK_NVRTC(err):
        if err != nvrtc.nvrtcResult.NVRTC_SUCCESS:
            raise RuntimeError('Nvrtc Error: {}'.format(err))
    
    empty_cplusplus_kernel = "__global__ void A() {}"
    err, program_handle = nvrtc.nvrtcCreateProgram(empty_cplusplus_kernel.encode(), b"", 0, [], [])
    CHECK_NVRTC(err)
    nvrtc.nvrtcCompileProgram(program_handle, 1, ["-dlto".encode()])
    err, size = nvrtc.nvrtcGetLTOIRSize(program_handle)
    CHECK_NVRTC(err)
    empty_kernel_ltoir = b" " * size
    err, = nvrtc.nvrtcGetLTOIR(program_handle, empty_kernel_ltoir)
    CHECK_NVRTC(err)
    err, = nvrtc.nvrtcDestroyProgram(program_handle)
    CHECK_NVRTC(err)
    return empty_kernel_ltoir
```

Reply:
@ksimpson-work I noticed this was marked as resolved but no change was made? Let's make sure we have changes checked in and pushed (or the review thread responded) before marking it resolved.